scripts/nginx.conf

```diff
@@ -0,0 +1,57 @@
+log_format balrog_admin_format
+    'admin $msec "$remote_addr" "$request" $status $request_length $body_bytes_sent '
+    '$remote_user $request_time $upstream_response_time "$http_user_agent" "$gzip_ratio"';
+log_format balrog_agent_format
```

Comment:
`balrog_agent_format` doesn't seem to be used, and I don't see a "custom entrypoint for balrog agent" in this file like prod does. Is that because agent is talking directly to balrogadmin-py3 ? Is there a downside to making it talk to nginx to be even more prod-like ?

Reply:
Good catch. Let me look into doing that, it seems like it should work OK, and would be beneficial while we're testing the auth0 integration work, come to think of it.